tests/Proto.Actor.Tests/EventStreamTests.cs

```diff
@@ -48,5 +49,18 @@ public void EventStream_OnlyReceiveSubscribedToEventTypes()
             eventStream.Publish("not an int");
             Assert.Equal(0, eventsReceived.Count);
         }
+
+        [Fact]
+        public void EventStream_CanSubscribeToSpecificEventTypes_Async()
+        {
+            var received = "";
+            var eventStream = new EventStream();
+            eventStream.Subscribe<string>(theString =>
+            {
+                received = theString;
+                Assert.Equal("hello", received);
```

Comment:
I don't think this assertion ever gets called - try changing it to "some random string" and it still passes